Update AlphaAnimals_CE_Patch_Resources_Leathers.xml

Breadbox2k19 opened the pull request:
Removed Galatross horn as a melee weapon.
Made Skysteel useable as a crafting material.

Breadbox2k19 committed:
Update AlphaAnimals_CE_Patch_Resources_Leathers.xml

Removed Galatross horn as a melee weapon.
Made Skysteel useable as a crafting material.

Juanfrank reviewed (changes requested):
Why would Gallatross Horn be erased?

Sky Steel according to the wiki is "A weird iron based compound found in a fallen asteroid. Stronger than normal steel, but a terrible insulator." 

Patched Steel values are <StuffPower_Armor_Sharp>1</StuffPower_Armor_Sharp> and <StuffPower_Armor_Blunt>1.5</StuffPower_Armor_Blunt> 

Therefore I think it should be higher than Steel.

Breadbox2k19 commented:
Gallatross horn is only removed as a useable melee weapon, it still exists as an item to sell, this is to bring it inline with CE’s policy of ‘Horns are too heavy as melee weapons’, much like Thrumbo and elephant horns.
Not to mention gallatross horn atm isn’t supported by CE and will spam red errors if equipped.

I’m honestly whatever regarding balancing of skysteel, any-value would do, but considering that steel also has 0 heat protection in CE, it simply makes skysteel a strictly better version of steel.

Juanfrank commented:
Gotcha. I'll test it locally then probably merge and edit the SkySteel values for a marginal buff compared to Steel.

Juanfrank committed:
Update AlphaAnimals_CE_Patch_Resources_Leathers.xml

Slightly improved values according to the SkySteel Lore. Added description to Gallatross Horn.
